ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.h

```diff
@@ -43,6 +43,7 @@ class BrowseOGCAPIFeatureService : public QObject
   Q_PROPERTY(QString errorMessage READ errorMessage WRITE setErrorMessage NOTIFY errorMessageChanged)
   Q_PROPERTY(QUrl featureServiceUrl READ featureServiceUrl NOTIFY urlChanged)
   Q_PROPERTY(QStringList featureCollectionList READ featureCollectionList NOTIFY featureCollectionListChanged)
+  Q_PROPERTY(bool loading READ loading NOTIFY loadingChanged)
```

Comment:
This can all be inferred from `m_featureLayer` which already has a `loadStatusChanged` signal that notifies. You can simply reuse that logic as needed.

Is you are going to keep this, I would be more specific. Presumably there are a few Loadable objects in play and "loading" is a well-known API term. I would make this `featureLayerLoading` or something like that if you keep it.